### ISSUE TITLE
Incompatible with latest Firebase version

### ISSUE THREAD

**Issue by nijs9** [juliansteenbakker/mobile_scanner#1197]
When building an iOS app with mobile_scanner 5.2.3 with the latest firebase_core package, there is a problem with pod install:

[!] CocoaPods could not find compatible versions for pod "GoogleDataTransport":
  In snapshot (Podfile.lock):
    GoogleDataTransport (= 10.1.0, ~> 10.0)

  In Podfile:
    firebase_crashlytics (from '.symlinks/plugins/firebase_crashlytics/ios') was resolved to 4.1.3, which depends on
      Firebase/Crashlytics (= 11.2.0) was resolved to 11.2.0, which depends on
        FirebaseCrashlytics (~> 11.2.0) was resolved to 11.2.0, which depends on
          GoogleDataTransport (~> 10.0)

    mobile_scanner (from '.symlinks/plugins/mobile_scanner/ios') was resolved to 5.2.3, which depends on
      GoogleMLKit/BarcodeScanning (~> 6.0.0) was resolved to 6.0.0, which depends on
        GoogleMLKit/MLKitCore (= 6.0.0) was resolved to 6.0.0, which depends on
          MLKitCommon (~> 11.0.0) was resolved to 11.0.0, which depends on
            GoogleDataTransport (< 10.0, >= 9.4.1)

Does anybody know there is a solution to bypass this error?

**Comment by spencer2k19** [comment]
Same issue from my side: 
CocoaPods could not find compatible versions for pod "GoogleDataTransport":
  In Podfile:
    firebase_messaging (from .symlinks/plugins/firebase_messaging/ios) was resolved to 15.1.3, which depends on
      Firebase/Messaging (= 11.2.0) was resolved to 11.2.0, which depends on
        FirebaseMessaging (~> 11.2.0) was resolved to 11.2.0, which depends on
          GoogleDataTransport (~> 10.0)

    mobile_scanner (from .symlinks/plugins/mobile_scanner/ios) was resolved to 5.2.3, which depends on
      GoogleMLKit/BarcodeScanning (~> 6.0.0) was resolved to 6.0.0, which depends on
        MLKitBarcodeScanning (~> 5.0.0) was resolved to 5.0.0, which depends on
          MLKitCommon (~> 11.0) was resolved to 11.0.0, which depends on
            GoogleDataTransport (< 10.0, >= 9.4.1)

**Comment by qwezey** [comment]
same issue here with nanopb

**Comment by faizan1249** [comment]
Same Issue From My Side.... kindly anybody share the solution.

**Comment by octobocto** [comment]
https://github.com/juliansteenbakker/mobile_scanner/issues/1165

**Comment by pglaum** [comment]
In the meantime you can set all firebase dependencies to the versions from Aug 27, before they introduced Firebase SDK 11.

For example:

```
cloud_firestore: 5.2.1
firebase_auth: 5.1.4
firebase_core: 3.3.0
firebase_messaging: 15.0.4
firebase_remote_config: 5.0.4
```

**Comment by burakJs** [comment]
Same issue for my project

```
[!] CocoaPods could not find compatible versions for pod "Firebase/Auth":
  In Podfile:
    Firebase/Auth

    firebase_auth (from `.symlinks/plugins/firebase_auth/ios`) was resolved to 5.3.1, which depends on
      Firebase/Auth (= 11.2.0)

CocoaPods could not find compatible versions for pod "Firebase/Messaging":
  In Podfile:
    Firebase/Messaging

    firebase_messaging (from `.symlinks/plugins/firebase_messaging/ios`) was resolved to 15.1.3, which depends on
      Firebase/Messaging (= 11.2.0)

CocoaPods could not find compatible versions for pod "GoogleDataTransport":
  In Podfile:
    firebase_messaging (from `.symlinks/plugins/firebase_messaging/ios`) was resolved to 15.1.3, which depends on
      Firebase/Messaging (= 11.2.0) was resolved to 11.2.0, which depends on
        FirebaseMessaging (~> 11.2.0) was resolved to 11.2.0, which depends on
          GoogleDataTransport (~> 10.0)

    mobile_scanner (from `.symlinks/plugins/mobile_scanner/ios`) was resolved to 5.2.3, which depends on
      GoogleMLKit/BarcodeScanning (~> 6.0.0) was resolved to 6.0.0, which depends on
        MLKitBarcodeScanning (~> 5.0.0) was resolved to 5.0.0, which depends on
          MLKitCommon (~> 11.0) was resolved to 11.0.0, which depends on
            GoogleDataTransport (< 10.0, >= 9.4.1)
```

**Comment by navaronbracke** [comment]
As mentioned above, this is a duplicate of https://github.com/juliansteenbakker/mobile_scanner/issues/1165
